GrocifyApp.UI/src/components/Login.jsx

```diff
@@ -20,43 +31,67 @@ const LoginForm = () => {
 
             if (response.ok) {
                 const t = await response.text();
-                
+
                 localStorage.setItem('token', t);
-                
+
                 console.log('Login successful');
 
                 navigate('/weatherforecast');
             } else {
                 const errorData = await response.json();
-                console.error('Login failed', errorData);
+                console.error('Login failed', errorData.errors[0]);
+
+                setErrorMessage(errorData.errors[0]);
             }
         } catch (error) {
             console.error('Error during login', error);
+
+            setErrorMessage(GenericConsts.Error);
+        } finally {
+            loginButton.classList.remove("loading");
         }
     };
 
     return (
-        <div>
-            <h2>Login</h2>
-            <form>
-                <label>Email:</label>
-                <input
-                    type="email"
-                    value={email}
-                    onChange={(e) => setEmail(e.target.value)}
-                />
-                <br />
-                <label>Password:</label>
-                <input
-                    type="password"
-                    value={password}
-                    onChange={(e) => setPassword(e.target.value)}
-                />
-                <br />
-                <button type="button" onClick={handleLogin}>
-                    Login
+        <div className="container-page">
+            <ReactSVG src={ReactLogo} />
+            <form className="input-form">
+                <div className="title title--xl">{LoginConsts.SignIn}</div>
+                <p className="text color-n500 login-desc">{LoginConsts.EnterDetails}</p>
+                <p id="error" className="text error">{errorMessage}</p>
+                <div className="input-box mt-3">
+                    <input className="input-text"
+                        type="email" placeholder="name@example.com"
```

Comment:
remove placeholder for inputs

Reply:
needed, we use placeholder to know if input has text or no, and then translate label.

![image](https://github.com/Catarina98/Grocify/assets/36730010/26ca3ea0-d457-4648-99cf-f17c35aaa626)